packages/teams-js/src/internal/communication.ts

```diff
@@ -207,7 +207,7 @@ export function sendMessageToParent(actionName: string, callback?: Function): vo
  * @internal
  * Limited to Microsoft-internal use
  */
-export function sendMessageToParent(actionName: string, args: any[], callback?: Function): void;
+export function sendMessageToParent(actionName: string, args: any[] | undefined, callback?: Function): void;
```

Comment:
>undefined [](http://example.com/codeflow?start=70&length=9)

This function was already handling the case where the parameter was possibly `undefined`, and now the type indicates that.